### ISSUE TITLE
Style props given an undefined value are output as 'undefined' in css

### ISSUE THREAD

**Issue by garth** [kettanaito/atomic-layout#79]
## What

```jsx
<Composition templateCols={undefined} />
```

will output

```css
grid-template-columns: undefined;
```

## Why

not checking for undefined values

## Expected behavior

css attributes for props with undefined or null values should not be output

**Comment by kettanaito** [comment]
Hi, @garth. Thanks for pointing this out.

I will filter out props that have no value, so they don't result into `undefined` in the generated CSS. Should merge this today.

**Comment by kettanaito** [comment]
Published under `1.4.3.` version.
